src/action-creators/undoArchive.js

```diff
@@ -21,8 +18,8 @@ export const undoArchive = ({ originalPath, currPath, offset }) => dispatch => {
 
   const { getState } = store
```

Comment:
Use the second argument to the thunk rather than importing the store. See other action-creators for this pattern.